gordo_client/io.py

```diff
@@ -76,8 +78,11 @@ def _handle_response(resp: requests.Response, resource_name: Optional[str] = Non
         In case of network or IO errors
     """
     if 200 <= resp.status_code <= 299:
-        is_json = resp.headers["content-type"] == "application/json"
-        return resp.json() if is_json else resp.content
+        if _is_json_response(resp):
+            return resp.json()
+        elif _is_azure_login_response(resp):
```

Comment:
`gordo-client` is a platform agnostic open-source client, so there is should not be any sanctions about azure
I suggest removing this elsif branch.